packages/core/src/service/CacheService.ts

```diff
@@ -219,7 +221,7 @@ export class CacheService {
 			filePath = await this.getCacheFileUri()
 			const cache: CacheFile = {
 				version: LatestCacheVersion,
-				projectRoot: this.project.projectRoot,
+				projectRoot: this.project.projectRoots[0],
```

Comment:
Can we store all roots here? I can't remember what is this used for though.

Reply:
Afaict this field isn't used